packages/flutter_tools/test/commands.shard/permeable/packages_test.dart

```diff
@@ -201,36 +201,6 @@ void main() {
       }
     }
 
-    testUsingContext('get fetches packages and has output from pub', () async {
```

Comment:
Test no longer possible because of the "pub get" command adopting ProcessStartMode.inheritStdio, which does not allow you to mock stdout/stderr.

Reply:
LGTM